data-prepper-plugins/mutate-string-processors/README.md

```diff
@@ -44,7 +44,8 @@ If `from` regex string does not have a match, the key will be returned as it is.
 ### Configuration
 * `entries` - (required) - A list of entries to add to an event
     * `source` - (required) - The key to be modified
-    * `from` - (required) - The Regex String to be replaced
+    * `from` - (required) - The Regex String to be replaced. Special regex characters such as `[` and `]` 
```

Comment:
This uses Java regex syntax. I think it would also help to note that and link to the Java regex syntax page.